keras/src/legacy/preprocessing/sequence.py

```diff
@@ -64,7 +66,9 @@ def __init__(
         shuffle=False,
         reverse=False,
         batch_size=128,
+        **kwargs,
     ):
+        super().__init__(**kwargs)
```

Comment:
I realize it's all deprecated code, but just for consistency, can you fix `get_config` line 152?

It needs the usual:
```python
        config = super().get_config()
        config.update({
            "data": json_data,
            ...
        })
        return config
```
to persist what came through `__init__`.